.github/workflows/CI.yml

```diff
@@ -70,6 +70,9 @@ jobs:
         with:
           version: ${{ matrix.julia-version }}
       - uses: julia-actions/cache@v1
+        with:
+          cache-name: julia-cache;workflow=${{ github.workflow }};julia=${{ matrix.julia-version }}
```

Comment:
I think we should include the os here. Afaik the compiled stuff is not necessarily compatible between Ubuntu and macos

Reply:
Ah ok. Merging caches is great!